### ISSUE TITLE
add background and assets in game folder 

### ISSUE THREAD

**Issue by otakuparshva** [otakuparshva/Platformer-game-website---2nd-year-internship-project-#11]
In the game folder, there is a `static` folder. Inside the `static` folder, there is an `assets` folder and a `game` folder. 

- In the `assets` folder, all asset files are stored.
- In the `game` folder, there is a `game.py` file which includes code for the background.

Here is the structure:

```
game/
│
├── static/
│   ├── assets/
│   │   └── [all asset files]
│   └── game/
│       └── game.py (includes background code)
```

---

Let me know if you need further clarification or additional information included.

**Comment by Meet-GU** [comment]
Working on #11

**Comment by Meet-GUP** [comment]
fixed #11
